# -*- coding: utf-8 -*-
# Generated by Django 1.10.5 on 2017-02-20 23:57
from __future__ import unicode_literals

from django.db import migrations, models


class Migration(migrations.Migration):

    dependencies = [
        ('vault', '0004_loginattempt_error_msg'),
    ]

    operations = [
        migrations.AddField(
            model_name='vaultuser',
            name='email',
            field=models.EmailField(default='', max_length=254),
        ),
        migrations.AddField(
            model_name='vaultuser',
            name='username',
            field=models.CharField(default='', max_length=255),
        ),
    ]
